model/distro/distro.go

```diff
@@ -45,7 +45,14 @@ func init() {
 
 // GenerateName generates a unique instance name for a distro.
 func (d *Distro) GenerateName() string {
-	return fmt.Sprintf("evg-%s-%s-%d", d.Id, time.Now().Format(evergreen.NameTimeFormat), rand.Int())
+	switch d.Provider {
+	case evergreen.ProviderNameStatic:
```

Comment:
Do we actually need anything other than the default case? I don't think we need static, since `GetInstanceName()` is called by the scheduler when it spawns hosts, so presumably `d.GenerateName()` will never be called for static hosts. I don't know enough about docker to argue convincingly that we don't need the docker case, but if we're generating our own ids, do they really need to start with "container"?.